### PR TITLE
print svelte warnings (fixes #3)

### LINKED ISSUE #3: Svelte's debug warnings are not shown
We need to log those to the console probably! We should check out how the `svelte` `webpack` and `rollup` loaders work.

### PR DESCRIPTION
**Pull request by v0idifier**
### Which issue does this fix?

Closes #3 

### Describe the solution

i looked at how `svelte-loader` does it. keep in mind that the way this is done here is not compatible with svelte v2, but we aren't supporting that, right?

also, we could add a little color to the output to highlight it over other things but i didn't want to bring in any other package.

### CODE REVIEW

**Comment by v0idifier** [comment]
it's breaking tests because i also passed the `filename` option to the svelte compiler to possibly print better warnings? and that outputs a different source map file in the test. what should i do?

File: src/index.ts [hunk 1]
```diff
@@ -19,15 +19,25 @@ async function compile(srcPath: string): Promise<string | null> {
...
+                dev: !IS_PRODUCTION_MODE,
+                hydratable: process.argv.includes('--hydratable'),
+                immutable: process.argv.includes('--immutable'),
+            });
+
+            warnings.forEach(warning => {
+                console.warn(
+                    `Warning from compiler in file ${warning.filename}: ${warning.message}\n${warning.frame}`
+                );
+            });
```
Comment (jakedeichert) [inline src/index.ts]: Brilliant! Glad this was so easy 🙌 

I'm going to try this out tonight :)

**Comment by jakedeichert** [comment]
> compatible with svelte v2, but we aren't supporting that, right?

Correct, only svelte 3 👍 

> also, we could add a little color to the output to highlight it over other things but i didn't want to bring in any other package.

Thank you for not adding a package in this PR!! I also would love colored log out, much better DX, and i'm going to investigate the ecosystem for a package. In accordance with one of the goals (Keep your `node_modules` light), i'm going to check if one of our deps already has a color logger that we can depend on the same version of. Very likely there is already one. But that'll be part of a bigger cleanup too.. i want the logging to be a bit cleaner and i have some ideas.

> it's breaking tests because i also passed the filename option to the svelte compiler to possibly print better warnings? and that outputs a different source map file in the test. what should i do?

I'll run it locally and take a look. Most likely we'll commit the new snapshot updates.

**Comment by jakedeichert** [comment]
Snapshot test failed because svelte updated, so the .js.map that snowpack generated was different. If this happens too frequently, i'll consider removing those files from the snapshot.

Ran this locally and it's great! :100:

**Pull request merged**